data/decks/jin.json

```diff
@@ -43,6 +43,12 @@
 		}
 	],
 	"exceed_ability_effects": [
+		{
+			"timing": "on_reshuffle",
```

Comment:
lol, I left out arctic dungeon?

Reply:
oh snap, if you're exceeded and reshuffle it's not there, haha, okay

Reply:
yee i doubt anyone will do this intentionally but